scrapi/tasks.py

```diff
@@ -102,16 +118,30 @@ def normalize(raw_doc, harvester_name):
     return normalized  # returns a single normalized document
 
 
-@app.task(default_retry_delay=300, max_retries=5)
+@app.task
 @events.logged(events.PROCESSING, 'normalized')
 def process_normalized(normalized_doc, raw_doc, **kwargs):
-    if not normalized_doc:
-        raise events.Skip('Not processing document with id {}'.format(raw_doc['docID']))
-    processing.process_normalized(raw_doc, normalized_doc, kwargs)
-
-
-@app.task(default_retry_delay=300, max_retries=5)
-def update_pubsubhubbub():
-    payload = {'hub.mode': 'publish', 'hub.url': '{url}rss/'.format(url=settings.OSF_APP_URL)}
-    headers = {'Content-Type': 'application/x-www-form-urlencoded'}
-    return requests.post('https://pubsubhubbub.appspot.com', headers=headers, params=payload)
+    try:
+        if not normalized_doc:
+            raise events.Skip('Not processing document with id {}'.format(raw_doc['docID']))
+        processing.process_normalized(raw_doc, normalized_doc, kwargs)
+    except Exception as e:
+        logger.info('Retrying withiin PROCESS NORMALIZED with exception {}'.format(e))
+        process_normalized.retry(exc=e)
+
+
+@app.task
+def migrate(migration, **kwargs):
+    count = 0
+    for doc in documents(kwargs.get('source')):
+        count += 1
+
+        if kwargs.get('async'):
+            migration.s(doc, **kwargs).apply_async()
+        else:
+            migration(doc, **kwargs)
+
+    if kwargs.get('dry'):
```

Comment:
This can probably be handled by the individual migration tasks, since some may not support dry runs.